Documentation/spelling_wordlist.txt

```diff
@@ -915,6 +915,7 @@ rebase
 rebased
 rebasing
 recompiles
+reconciliationTriggerInterval
```

Comment:
We need to stop this madness, I've created https://github.com/cilium/cilium/pull/26759. But OK for this PR :+1:

Reply:
Sorry, #26759 was merged and created a conflict for your PR :grimacing:. Can you please rebase, re-generate the Helm value update, and remove this new entry from the `spelling_wordlist.txt`?

Reply:
Updated :heavy_check_mark: